include/ibamr/IBHydrodynamicSurfaceForceEvaluator.h

```diff
@@ -190,12 +222,25 @@ class IBHydrodynamicSurfaceForceEvaluator
     double d_surface_contour_value = 0.0;
 
     /*!
-     * \brief File streams associated for the output.
+     * \brief Whether to write results on a text file.
+     */
+    bool d_write_to_file = false;
+
+    /*!
+     * \brief File streams associated for the output of hydrodynamic force.
```

Comment:
As noted above, lets remove this and the other logging in favor of a method taking a `std::ostream &` argument.